tests/acceptance-tests/test/specs/guest-authorizenet-checkout.app-test.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+let yaml = require("js-yaml");
+let fs   = require("fs");
+let expect = require("chai").expect;
+let shopUser = require("../../lib/user-shop-actions.js");
+
+beforeEach(function () {
+  let browserConfig = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/config/settings.yml", "utf8"));
+  const baseUrl = browserConfig.base_url.toString();
+  browser.url(baseUrl);
+});
+
+
+describe("authorize net guest checkout test", function () {
+  let eleMap = yaml.safeLoad(fs.readFileSync("./tests/acceptance-tests/elements/element-map.yml", "utf8"));
+  it("verify guest can checkout with authorize net", function () {
+    browser.pause("5000");
```

Comment:
Do we really need to have all these 5 second pauses? There is no way to just wait for the appropriate element to appear? Seems like this will take forever to run